ize/ize.mk

```diff
@@ -1,13 +1,17 @@
 #### IZE Download script
-CMD_DOWNLOAD_IZE = curl -s $(IZE_DOWNLOAD_URL) -o "$(TMPDIR)/$(OS_NAME).zip" && unzip -qq -o "$(TMPDIR)/$(OS_NAME).zip" -d "$(IZE_DIR)" && rm "$(TMPDIR)/$(OS_NAME).zip"
+CMD_DOWNLOAD_IZE = curl -L $(IZE_DOWNLOAD_URL) -o $(TMPDIR)/$(IZE_ARCHIVE_NAME) && tar -xvzf  $(TMPDIR)/$(IZE_ARCHIVE_NAME) -C $(IZE_DIR) && rm $(TMPDIR)/$(IZE_ARCHIVE_NAME)
 
 #### IZE Install script
 CMD_INSTALL_IZE =  chmod +x $(IZE_DIR)/ize
 
+#### Create IZE folder
+CMD_CREATE_IZE_FOLDER = mkdir -p $(IZE_DIR)
```

Comment:
@kobrikx do we really need a variable for this command? I think we can use it as  it is in the `ize.install` target